public/style.css

```diff
@@ -93,3 +93,38 @@ input {
     font-family: var(--fontMono);
     white-space: nowrap;
 }
+
+/** Recommendation Components **/
+.recommendation-wrapper {
+    min-width: 706px;
+    width: 100%;
+}
+
+.recommendation-node-wrapper {
+    padding: 20px;
+}
+
+.routing-node-address {
+    font-size: .8em;
+    color: #105fb0;
+}
+
+.recommendation-node-list {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+}
+
+.recommendation-node-property {
+    font-weight: 1000;
+    font-size: 1.1em;
+    text-shadow: 10px 10px 15px rgb(0, 0, 0);
+    padding: 0px 0px 0px 10px;
+    color: whitesmoke;
+    background-clip: text;
+    -webkit-background-clip: text; 
+}
+
+.recommendation-node-button-wrapper {
+    padding: 0px 0px 0px 10px; 
+}
```

Comment:
Why is everything custom? Shouldn't we be using the framework theme we're using and modifying it if we want changes? 

https://selekkt.dk/skelet/v3/

Reply:
I actually didnt know we had a css framework. We should put a mention in the README or contributing.md.